test/sanity/validate-modules/ignore.txt

```diff
@@ -145,75 +145,7 @@ lib/ansible/modules/cloud/atomic/atomic_container.py E317
 lib/ansible/modules/cloud/atomic/atomic_container.py E326
 lib/ansible/modules/cloud/azure/_azure.py E324
 lib/ansible/modules/cloud/azure/_azure.py E326
-lib/ansible/modules/cloud/azure/azure_rm_acs.py E324
-lib/ansible/modules/cloud/azure/azure_rm_acs.py E325
-lib/ansible/modules/cloud/azure/azure_rm_acs.py E326
-lib/ansible/modules/cloud/azure/azure_rm_availabilityset.py E324
-lib/ansible/modules/cloud/azure/azure_rm_availabilityset_facts.py E322
 lib/ansible/modules/cloud/azure/azure_rm_containerinstance.py E324
```

Comment:
why containerinstance not fixed?

Reply:
because test is failing, so i decided to fix it separately